src/System.IO.Ports/src/Interop/Unix/Interop.Termios.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO.Ports;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class Termios
+    {
+        internal enum Signals
+        {
+            SignalDtr = 1,
```

Comment:
@stephentoub, what is the convention for enums shared between native and managed code - should the names of the members be prefixed by the "Signal" even on the managed side?
And, should the name of the enum type really be plural?

Reply:
> should the names of the members be prefixed by the "Signal" even on the managed side?

In most cases we name the enum value the same as the native symbol and we name the enum something meaningful / descriptive, e.g.
```C#
        [Flags]
        internal enum MemoryMappedProtections
        {
            PROT_NONE = 0x0,
            PROT_READ = 0x1,
            PROT_WRITE = 0x2,
            PROT_EXEC = 0x4
        }
```